app/webpacker/styles/lead_providers.scss

```diff
@@ -0,0 +1,152 @@
+/* ==========================================================================
+   #INDEX GROUP
+   ========================================================================== */
+
+.index-group {
+  background: #f5f5f5;
+  padding: 1.5em;
+  margin-bottom: 1.5em;
+}
+.index-group p.govuk-body {
+  margin-bottom: 0;
+}
+.index-group .govuk-heading-m {
+  margin-bottom: 10px
+}
+
+/* ==========================================================================
+   #SECONDARY NAV
+   ========================================================================== */
+.hmcts-sub-navigation {
```

Comment:
Like I mentioned, I do wonder if we need both callums `tabs` and this sub-navigation, I think they look very much similar. Happy for you to make a call on that though.

Reply:
Is this code taken from somewhere else? If so, this project used to have (and ecf-engage-and-learn now has) a vendor_styles directory that might be worth recreating for this.

https://github.com/DFE-Digital/ecf-engage-and-learn/tree/develop/app/webpacker/vendor_styles

Reply:
https://trello.com/c/AW5RtUGq/317-recreate-vendor-styles-based-on-engage-and-learn-from-callum-comments

Created ticket for that - for now will move those styles to its own directories

Reply:
Done @kicferk1